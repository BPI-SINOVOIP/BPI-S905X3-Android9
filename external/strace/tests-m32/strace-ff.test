#!/bin/sh

# Check -ff option.

. "${srcdir=.}/init.sh"

run_prog_skip_if_failed \
	kill -0 $$

../set_ptracer_any ../sleep 3 > "$OUT" &
tracee_pid=$!

while ! [ -s "$OUT" ]; do
	kill -0 $tracee_pid 2> /dev/null ||
		fail_ 'set_ptracer_any sleep failed'
done

run_strace -a14 -eexit_group -ff -p $tracee_pid

# check that output matches
match_diff "$LOG.$tracee_pid"

# check that no other output files have been created
set -- "$LOG".*
[ "$LOG.$tracee_pid" = "$*" ] ||
	fail_ "too many output files: $*"
