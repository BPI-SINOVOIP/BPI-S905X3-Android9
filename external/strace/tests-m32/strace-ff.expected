exit_group(0) = ?
+++ exited with 0 +++
